schema.ts

```diff
@@ -1232,6 +1232,80 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
   // TODO(dweitzman): Only support options in the typing when the codaType is ValueHintType.SelectList.
 }
 
+/**
+ * Discriminent used to determine the permission type
+ * @hidden
+ */
+export enum PermissionType {
+  User = 'user',
+  Group = 'group',
+  Domain = 'domain',
+  Public = 'public',
```

Comment:
What does this actually mean? For our current use case, this will be "organization-wide", should that be the principal that makers actually specify, given that?

Reply:
This is a big question I had. I think the problem is ingestions can be configured in some context (that we have not decided upon fully. Could be workspaces or organizations). Does it make sense for the pack maker to specify which context this thing is available in. I think what they are trying to communicate is `This is available to anyone who has access to the sync`. Happy to discuss